front/app/containers/PlatformFooter/ShortFeedback/index.tsx

```diff
@@ -0,0 +1,252 @@
+import React, { useState } from 'react';
```

Comment:
Extracted this from PlatformFooter. Still lots to improve in the component, but not worth the time investment atm as these files are barely touched.